### ISSUE TITLE
sumo issue:

### ISSUE THREAD

**Issue by iamxg** [intrig-unicamp/mininet-wifi#158]
Hi Ramon,
How are you?
The command "py car1.params['position']" would return (0,0,0) when I deleted this line   <remote-port value="8813"/> in configure file "map.sumocfg". But time in sumo-gui so slowly changed when I added this line   <remote-port value="8813"/>.  the file map.sumocfg is as follows. Could you tell me where this issue may be?
I am so sorry for disturbing you. Thanks a lot!
Xian Guo

?xml version="1.0" encoding="UTF-8"?>

<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/sumoConfiguration.xsd">

    <input>
        <net-file value="osm.net.xml"/>
        <route-files value="osm.rou.xml"/>
        <additional-files value="osm.poly.xml"/>
    </input>

    <output>
        <tripinfo-output value="maptripinfo.out.xml"/>
    </output>

    <time>
        <begin value="0"/>
        <end value="1000"/>
        <step-length value="0.1"/>
    </time>

    <report>
        <verbose value="true"/>
        <duration-log.statistics value="true"/>
        <no-step-log value="true"/>
    </report>

    <traci_server>
         <processing>
                <time-to-teleport value="100"/>
         </processing>
         <remote-port value="8813"/>
    </traci_server>
   <gui_only>
        <gui-settings-file value="osm.view.xml"/>
    </gui_only>
</configuration>

**Comment by ramonfontes** [comment]
Dear Guo,

Could you provide more detail about the issue and what those XML lines
mean? What is the relationship between the issue and those lines?

**Comment by iamxg** [comment]
Hi Ramon,
So sorry, I just found that I didn't put this line that may be relative with my issue. 

This XML lines are lines in the configuration file "map.sumocfg". 

The command "py car1.params['position']" would return (0,0,0) when I deleted this line "<remote-port value="8813"/>" in the configure file "map.sumocfg". 

But the frequent of “time” addition is very slow in Sumo-GUI when I added this line. That is to say, the time value will increase 0.1 after a few seconds, which result in the simulation speed is very slow.

I try to simulate 200 vehicles. I set different delay values in Sumo-GUI parameter dialog. But it didn't work.
I don't know to resolve it.
